learn/safe-core/safe-core-account-abstraction-sdk/onramp-kit.md

```diff
@@ -1,115 +1,209 @@
-# ⚠️ Warning ⚠️
+# OnRamp Kit
+
+The [Onramp kit](https://github.com/safe-global/account-abstraction-sdk/tree/main/packages/onramp-kit) allows users to buy cryptocurrencies using a credit card and other payment options.
+
+## ⚠️ Warning ⚠️
 
 This package is provided for testing purposes only. It's not ready for production use. We are working with Stripe and participating in the pilot test for their new [crypto on-ramp](https://stripe.com/docs/crypto). Considering this, we provide a public key and a testing server already configured during the [Safe Account Abstraction hackathon](https://safe-global.notion.site/Safe-d6c6ed61389041e28f5c7c925f653701)
 
-Once the hackathon and Stripe pilot are over, the server will be removed and you should use your own keys and server in case you opt-in for the [StripeAdapter](https://github.com/safe-global/account-abstraction-sdk/blob/838d89e98aa9f9e32a6cd499a898fa7f6e69e7c6/packages/onramp-kit/src/adapters/stripe/StripeAdapter.ts).
+Once the hackathon and Stripe pilot are over, the server will be removed and you should use your own keys and server if you plan on using the [StripeAdapter](https://github.com/safe-global/account-abstraction-sdk/blob/838d89e98aa9f9e32a6cd499a898fa7f6e69e7c6/packages/onramp-kit/src/adapters/stripe/StripeAdapter.ts).
+
+## Quickstart
 
-Currently this package is only prepared to work with Stripe. See [considerations and limitations](#considerations-and-limitations) for more details.
+In this tutorial, we'll create a React component that allows you to enter an Ethereum address and fund it using a credit card. The full code example can be found in the [Safe Space repo](https://github.com/5afe/safe-space) in the [WalletFund.tsx file](https://github.com/5afe/safe-space/blob/onramp-kit-integration/src/scenes/Wallet/WalletFund.tsx) in [PR #8](https://github.com/5afe/safe-space/pull/8). We will also be using a [pre-deployed server](https://github.com/5afe/aa-stripe-service).
 
-# OnRamp Kit
+### Demo
+https://user-images.githubusercontent.com/9806858/228415245-7b18b226-5ef4-4b7d-a4b1-5c9c53f625f5.mp4
 
-The [Onramp kit](https://github.com/safe-global/account-abstraction-sdk/tree/main/packages/onramp-kit) allows users to buy cryptocurrencies using a credit card and other payment options.
+<img width="394" alt="Screen Shot 2023-03-29 at 4 34 44 AM" src="https://user-images.githubusercontent.com/9806858/228415363-024fbfcd-443e-4024-8a2f-56390a09e9d4.png">
+
+<img width="388" alt="Screen Shot 2023-03-29 at 4 37 04 AM" src="https://user-images.githubusercontent.com/9806858/228415381-cf838ccc-4002-4327-a654-7c9dc2ba68f7.png">
+
+<img width="748" alt="Screen Shot 2023-03-29 at 4 36 52 AM" src="https://user-images.githubusercontent.com/9806858/228415378-901931d8-7095-4257-a7ae-5ff67246c297.png">
```

Comment:
Alt should be relevant information describing the image to help screen readers

Reply:
Thanks for the catch. Fixed: https://github.com/safe-global/safe-docs/pull/192/commits/9f225d29932aa7eb56fd4ed54fa74bf600f19228